x/ecocredit/go.mod

```diff
@@ -0,0 +1,26 @@
+module github.com/regen-network/regen-ledger/x/ecocredit
+
+go 1.15
+
+require (
+	github.com/btcsuite/btcutil v1.0.2
+	github.com/cockroachdb/apd/v2 v2.0.2
+	github.com/cosmos/cosmos-sdk v0.42.4
+	github.com/enigmampc/btcutil v1.0.3-0.20200723161021-e2fb6adb2a25
+	github.com/gogo/protobuf v1.3.3
+	github.com/gorilla/mux v1.8.0
+	github.com/grpc-ecosystem/grpc-gateway v1.16.0
+	github.com/regen-network/regen-ledger/orm v0.0.0-00010101000000-000000000000
+	github.com/regen-network/regen-ledger/types v0.0.0-00010101000000-000000000000
+	github.com/spf13/cobra v1.1.3
+	github.com/spf13/pflag v1.0.5
+	github.com/stretchr/testify v1.7.0
+	google.golang.org/grpc v1.37.0
+	gopkg.in/yaml.v2 v2.4.0
+)
+
+replace github.com/regen-network/regen-ledger/orm => ../../orm
+
+replace github.com/regen-network/regen-ledger/types => ../../types
+
+replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
```

Comment:
Should we be consistent on whether relative replaces go under or above absolute replaces? (I prefer relative ones unders)